docs/Installation/Installation_Guide.rst

```diff
@@ -21,74 +21,68 @@ public cloud resources, the following sizing requirements are assumed good for
 other cloud Kubernetes deployments, where such resources are preinstalled and
 available on demand.
 
-####################
 Setting Up a Cluster
 ####################
 
-****************************
 General Cluster Requirements
 ****************************
 
 Setting up a testing cluster requires at least three machines (these can be
-VMs) running CentOS_ 7.4 (The recommended mimimum for Zenko production service
-is five server nodes with three masters/etcds, but for testing and
-familiarization, three masters and three nodes is fine). You must have SSH
+VMs) running CentOS_ 7.4 (or higher)(The recommended mimimum for Zenko
+production service is five server nodes with three masters/etcds, but for
+testing and familiarization, three masters and three nodes is fine). You must
+have SSH
 access to these machines and they must have SSH access to each other. (You
 can copy SSH credentials from one machine to the next and log in once to
 ensure each machine has been added to the others' recognized hosts lists).
 Each machine acting as a Kubernetes_ node must also have at least one disk
 available to provision storage volumes.
 
+Once you have set up a cluster, you cannot change the size of the machines on
+it.
+
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _CentOS: https://www.centos.org
 .. _Kubernetes: https://kubernetes.io
 
-
 Sizing
 ======
 
-Sizing for Metal K8s
---------------------
-
--  Cores
+The following sizes have been for Zenko instances tested on live systems using
+MetalK8s, which adds some overhead. If you are running a different Kubernetes
+engine, fewer resources may be required, but such configurations remain to be
+tested.
 
-   -  2 for etcd
-   -  4 for master
-   -  4 for node
+Reserve the following resources for each node.
 
--  RAM
+-  Cores per server (2 vcpu = 1 core)
 
-   -  4 GB for etcd
-   -  8 GB for master
-   -  8 GB for node
+   - 12 cores minimum
```

Comment:
These are vCPU and should be 
24 vCPU minimum
32 vCPU medium load
58 vCPU heavy load

Reference: https://docs.google.com/presentation/d/1cKNQKa5YlnCKYIOjCOy4KeM9oZbWlLuAZbxCTF2uqN0/edit#slide=id.g3fd7445d35_1_5

Reply:
changed and pushed.

Reply:
Or rather get the cores too from the link